proto/scene-format.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package com.github.computer_graphics_course;
+
+message Scene {
+	enum MaterialType {
+		SOLID = 1;
+		GLASS = 2;
+		// other materials
+	}
+	message Vector3 {
+		double x = 1;
+		double y = 2;
+		double z = 3;
+	}
+	message Color {
+		double r = 1;
+		double g = 2;
+		double b = 3;
+	}
+	message Transform {
+		int32 id = 1;
+		Vector3 position = 2;
+		Vector3 rotation = 3;
+		Vector3 scale = 4;
+		int32 parent_id = 5;
+	}
+	message Cube {
+		int32 id = 1;
+		Vector3 upper_bounds = 2;
+		Vector3 lower_bounds = 3;
+	}
+	message Sphere {
+		int32 id = 1;
+		double radius = 2;
+	}
+	message Plain {
+		int32 id = 1;
+		Vector3 positon = 2;
+		Vector3 normal = 3;
```

Comment:
As mentioned previously, we do not need these two fields because the plane is completely defined by transform.